.github/workflows/integration.yml

```diff
@@ -28,10 +24,11 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-go-
       - run: sudo apt-get update -y && sudo apt-get install -y expect
-      - run: git config --global url."https://${{ secrets.GH_ACCESS_TOKEN }}@github.com".insteadOf "https://github.com"
-      - run: go env -w GOPRIVATE="github.com/bnb-chain/*"
       # used to debug workflow
 #      - name: Setup tmate session
 #        uses: mxschmitt/action-tmate@v3
       - run: make integration_test
       - run: make test
+      - run: make test_coverage
```

Comment:
If we get `test_coverage`, do we still need `make test`?

Reply:
There is a deadlock detection in the original `make test`, not covered by `make test_coverage`. So I left the original one here.

```
test: get_tools
	make set_with_deadlock
	make test_unit
	make test_race
	make cleanup_after_test_with_deadlock
```